deploy/upgrades/canaries/clustergroupupgrade.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: ran.openshift.io/v1alpha1
 kind: ClusterGroupUpgrade
 metadata:
   name: canaries
-  namespace: default
+  namespace: openshift-cluster-group-upgrades
```

Comment:
Why you change all the namespaces of the CRs for testing?
The 'openshift-cluster-group-upgrades' namespace is supposedly where the controller will be placed unto, but the user can and should be able to open ClusterGroupUpgrade CRs on any namespace.